eigrpd/eigrp_interface.c

```diff
@@ -377,6 +373,109 @@ bool eigrp_if_is_passive(struct eigrp_interface *ei)
 	return true;
 }
 
+
+/* Utility function for looking up passive interface settings. */
+static int eigrp_passive_nondefault_lookup(struct eigrp *eigrp, const char *ifname)
+{
+	unsigned int i;
+	char *str;
+
+	for (i = 0; i < vector_active(eigrp->passive_nondefault); i++) {
+		str = vector_slot(eigrp->passive_nondefault, i);
+		if (str && !strcmp(str, ifname))
+			return i;
+	}
+	return -1;
```

Comment:
I see this function could be converted to boolean, right?

Reply:
Nope, it returns the interface index within a vector or -1 if an interface is missing.

Reply:
there is a definition for an invalid interface ... I'll find it and put it here, but that's what we should use rather than -1

Reply:
IFINDEX_INTERNAL